selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/inject/SeleniumTestHandler.java

```diff
@@ -464,6 +470,28 @@ private void captureScreenshotsFromOpenedWindows(
             });
   }
 
+  private void storeWebDriverLogs(ITestResult result) {
+    Set<SeleniumWebDriver> webDrivers = new HashSet<>();
+    Object testInstance = result.getInstance();
+    collectInjectedWebDrivers(testInstance, webDrivers);
+    webDrivers.forEach(webDriver -> storeWebDriverLog(result, webDriver));
+  }
+
+  private void storeWebDriverLog(ITestResult result, SeleniumWebDriver webDriver) {
+    try {
+      String testReference = getTestReference(result);
+      String filename = NameGenerator.generate(testReference + "_", 4) + ".log";
+      Path webdriverLogsDirectory = Paths.get(webDriverLogsDir, filename);
+      Files.createDirectories(webdriverLogsDirectory.getParent());
+      Files.write(
+          webdriverLogsDirectory,
+          WebDriverLogsReader.getCombinedLogs(webDriver).getBytes(Charset.forName("UTF-8")),
+          StandardOpenOption.CREATE);
+    } catch (WebDriverException | IOException e) {
+      LOG.error(format("Can't get of the logs from WebDriver for test %s", result), e);
```

Comment:
`LOG.error(format("Can't get of the logs from WebDriver for test %s", result), e);`
>
`LOG.error("Error of storing web driver logs of test {}: {}", testReference, e);`